examples/experimental/dagster-airlift/dagster_airlift/core/sensor/builder.py

```diff
@@ -57,9 +58,11 @@ def check_keys_for_asset_keys(
 def build_airflow_polling_sensor_defs(
     airflow_instance: AirflowInstance,
     airflow_data: AirflowDefinitionsData,
-    event_translation_fn: AirflowEventTranslationFn,
-    minimum_interval_seconds: int = DEFAULT_AIRFLOW_SENSOR_INTERVAL_SECONDS,
+    event_translation_fn: AirflowEventTranslationFn = get_asset_events,
```

Comment:
for similar reasons you will probably just want to make this `None`. The moment we add a function above or below it will you probably switch it to `None` to avoid encoding the default multiple times.

Also given that `get_asset_events` is technically a mutable reference object (everything is an object in Python after all) making this `None` is safer.